webservice/templates/visualizer_select_base.html

```diff
@@ -118,19 +125,43 @@ <h1 style="text-align: center;"><span style="font-weight: bold;">SeeK-path</span
 	  <label for="fileformat">Select here the file format:</label>
 	</div> 
 	<div class='col-sm-6'>
-	  <select name="fileformat">
+	  <select name="fileformat" id="fileformatSelect">
             <option value="qe-inp">Quantum ESPRESSO input</option>
             <option value="vasp">VASP POSCAR</option>
             <option value="xsf">XCrySDen (.xsf)</option>
             <option value="castep">CASTEP (.cell)</option>
-            <option value="pdb">Protein Data Bank format (.pdb)</option>
+	    <option value="pdb">Protein Data Bank format (.pdb)</option>
+	    <option value="xyz">XYZ File (.xyz)</option>
 	  </select>
 	</div> 
       </div>
-    <div class='row' style="text-align:center">
-      <p class='small'>By continuing, you agree with the <a href="../termsofuse/" target="_blank">terms of use</a> of this service.</p>
-      <input type="submit" value="Calculate my structure" class="btn btn-default">
-    </div>
+      <div id="xyzFormatFields" class="row" style="display: none">
+        <div class='col-sm-6'>Cell vectors</div>
```

Comment:
Can you specify the units? "Cell vectors (Å)"

Reply:
sure